Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```diff
@@ -0,0 +1,81 @@
+- type: entity
+  name: Mk32 "Universal"
+  parent: [BaseWeaponPistol, BaseC1Contraband]
+  id: WeaponPistolUniversal
+  description: The Mark 32 Offensive Handgun, produced by NanoTrasen's Small Arms Division. Uses .45 magnum ammo. # DeltaV - update Mk32
```

Comment:
Misleading description: It says the gun uses .45 Magnum, when in reality, it uses .35 pistol cartridges. This raises the question: What is the purpose of this pistol? How does it differ from the Mk58 (assuming it was meant to use .35 pistol ammo)? How does it compare to the N1984 (assuming it was designed for .45 pistol ammo)?

Reply:
As far as it goes on DeltaV, it does use .45 magnum, and is intended to be a powerful pistol intended for logistics (cargo) to purchase to fight nukies/dragons/zombies. Comes in a crate of 2

Reply:
gun_chamber:
        name: Chamber
        startingItem: CartridgePistol
        priority: 1
        whitelist:
          tags:
            - CartridgePistol

CartridgePistol is .35, not .45

Reply:
https://github.com/DeltaV-Station/Delta-v/blob/master/Resources/Prototypes/Nyanotrasen/Entities/Objects/Weapons/Guns/Pistols/pistols.yml#L35
It uses .45 in DeltaV, might have been modified by this PR.